src/meta-client/src/client.rs

```diff
@@ -60,6 +64,7 @@ pub struct MetaClientBuilder {
     enable_store: bool,
     enable_lock: bool,
     enable_procedure: bool,
+    enable_access_cluster_info: bool,
```

Comment:
I really hate so many options,  are they necessary? If they are always `true` or `false`, and the user will not change them at all, please eliminate them.

Reply:
In frontend, it is always true, but in datanode it is always false

Reply:
Got it. I think we can refactor all these options by using `Role`. But it's not in this PR.